zendesk/ticket_test.go

```diff
@@ -38,7 +38,7 @@ func TestTicketCRUD(t *testing.T) {
 	require.Equal(t, created.Tags, found.Tags)
 
 	input := Ticket{
-		Status: String("solved"),
+		Status:                  String("solved"),
```

Comment:
strange white space. Please `go-lint` :)

Reply:
It is used to align with `[]interface{}{"email3@example.com"}` below. I used `gofmt`and `make lint` does not complain. Do you really want me to revert this whitespace?

Reply:
Ooops. Sorry that makes sense :grimacing: Ignore